packages/@uppy/xhr-upload/src/index.js

```diff
@@ -383,7 +383,50 @@ module.exports = class XHRUpload extends Plugin {
       }).then((res) => {
         const token = res.token
         const host = getSocketHost(file.remote.companionUrl)
-        const socket = new Socket({ target: `${host}/api/${token}` })
+        const socket = new Socket({ target: `${host}/api/${token}`, autoOpen: false })
+
+        this.onFileRemove(file.id, () => {
+          socket.send('pause', {})
+          queuedRequest.abort()
+          resolve(`upload ${file.id} was removed`)
+        })
+
+        this.onPause(file.id, (isPaused) => {
```

Comment:
I thought xhr uploads can’t be pause-resumed? Or is this for `aws-s3-multipart` and such, that use xhr-upload?

Reply:
I think this is a copy-paste mistake :sweat_smile: